metricbeat/module/ceph/_meta/docs.asciidoc

```diff
@@ -0,0 +1,9 @@
+== ceph Module
+
+This is the ceph Module.
+
+We used code/reference from:
+https://github.com/influxdata/telegraf/blob/master/plugins/inputs/ceph/ceph.go
```

Comment:
I think we need to be careful about the licenses here. Getting inspired by some code is one thing, copy / pasting it an other. If we have the same code, we should use it as a library ;-)